android/src/main/java/com/swmansion/gesturehandler/core/FlingGestureHandler.kt

```diff
@@ -3,15 +3,16 @@ package com.swmansion.gesturehandler.core
 import android.os.Handler
 import android.os.Looper
 import android.view.MotionEvent
+import android.view.VelocityTracker
+import kotlin.math.hypot
 
 class FlingGestureHandler : GestureHandler<FlingGestureHandler>() {
   var numberOfPointersRequired = DEFAULT_NUMBER_OF_TOUCHES_REQUIRED
   var direction = DEFAULT_DIRECTION
 
   private val maxDurationMs = DEFAULT_MAX_DURATION_MS
-  private val minAcceptableDelta = DEFAULT_MIN_ACCEPTABLE_DELTA
-  private var startX = 0f
-  private var startY = 0f
+  private val minVelocity = DEFAULT_MIN_VELOCITY
+  private val minDirectionAlignment = DEFAULT_MIN_DIRECTION_ALIGNMENT
```

Comment:
```suggestion
  private val minDirectionalAlignment = DEFAULT_MIN_DIRECTIONAL_ALIGNMENT
```

Dunno, the adjective looks nicer to me. I can be talked out of it though.

Reply:
Applied.